app/views/mediatorConfigModal.html

```diff
@@ -15,19 +15,19 @@ <h4 class="modal-title" id="addMediatorConfigLabel">
     <fieldset>
 
       <div class="form-group-2columns" ng-repeat="def in mediator.configDefs">
-        <div class="form-group" style="float: left;">
+        <div class="form-group" style="float: left; width: 33%;">
```

Comment:
It's better practise to not use inline styles. I know this was originally my code, but maybe we should switch to a horizontal-form instead of hard coding changed to the bootstrap styles - http://getbootstrap.com/css/#forms-horizontal

(I learnt a fair bit doing the transactions refactor, I didn't know about this stuff before...)